docs/features/kubernetes/configuration.md

```diff
@@ -159,7 +159,12 @@ auth:
         audience: ${AUTH_OKTA_AUDIENCE}
 ```
 
-The following values are supported out-of-the-box by the frontend: `google`, `microsoft`, `okta`, `onelogin`.
+The following values are supported out-of-the-box by the frontend: `google`, `microsoft`,
+`okta`, `onelogin`.
+
+Take note that `oidcTokenProvider` is just the issuer for the token, you can use any
+of these with an OIDC enabled cluster, like using `microsoft` as the issuer for a GKE
+cluster.
```

Comment:
GKE clusters actually don't use the "OpenID Connect Tokens" auth strategy by default -- they use access tokens, so it's probably "Webhook Token Authentication" under the covers. You might need to use a feature called ["Identity Service for GKE"](https://cloud.google.com/kubernetes-engine/docs/how-to/oidc) in order for the actual "OpenID Connect Token" strategy to be used.

EKS clusters can be configured with an OIDC provider, which does actually use the "OpenID Connect Tokens" auth strategy under the covers. Maybe that's a better example? At least it doesn't require the use of a special named feature like "Identity Service"...

Reply:
Yes, will change